docs/references/ic-interface-spec.md

```diff
@@ -594,19 +594,19 @@ Users have the ability to learn about the hash of the canister's module, its cur
 
 The concrete mechanism that users use to send requests to the Internet Computer is via an HTTPS API, which exposes four endpoints to handle interactions, plus one for diagnostics:
 
--   At `/api/v2/canister/<effective_canister_id>/call` the user can submit update calls that are asynchronous and might change the IC state.
```

Comment:
This PR should be merged when the roll-out is complete and changelog should be updated before merging.